bindings-wasm/processHeat/losses/losses.cpp

```diff
@@ -255,21 +267,21 @@ EMSCRIPTEN_BINDINGS(solidLoadChargeMaterial) {
 
     register_vector<SolidLoadChargeMaterial>("SolidLoadChargeMaterialV");
 }
-// wallLosses
-EMSCRIPTEN_BINDINGS(wallLosses) {
-    class_<WallLosses>("WallLosses")
-        .constructor<>()
-        .constructor<double, double, double, double, double, double, double>()
-        .function("totalHeatLoss", &WallLosses::totalHeatLoss)
-        .function("id", &WallLosses::id)
-        .function("surfaceDescription", &WallLosses::surfaceDescription)
-        .function("shapeFactor", &WallLosses::shapeFactor)
-        .function("setID", &WallLosses::setID)
-        .function("setSurfaceDescription", &WallLosses::setSurfaceDescription)
-        .function("setShapeFactor", &WallLosses::setShapeFactor);
 
-    register_vector<WallLosses>("WallLossesV");
+// wall_heat_loss
+EMSCRIPTEN_BINDINGS(wall_heat_loss) {
+    value_object<wall_heat_loss::ShapeFactor>("ShapeFactor")
+        .field("surfaceConfiguration", &wall_heat_loss::ShapeFactor::surface_configuration)
+        .field("value", &wall_heat_loss::ShapeFactor::value);
+
+    register_vector<wall_heat_loss::ShapeFactor>("ShapeFactorsVector");
+
+    function("shapeFactors", &wall_heat_loss::shapeFactors);
+    function("totalHeatLoss", &wall_heat_loss::totalHeatLoss);
```

Comment:
I believe we are going to need a more unique name for the binding than `totalHeatLoss`, possibly something like `totalWallHeatLoss`. I'm not sure if we want to also make the `convectiveHeatLoss` and `radiativeHeatLoss` functions more unique to walls as well. Most of the losses calculations will have a "totalHeatLoss" function that will need a unique binding name to differentiate them.